src/eterna/AnnotationManager.ts

```diff
@@ -721,16 +710,6 @@ export default class AnnotationManager {
         }
     }
 
-    public updateDialogLayers(): boolean {
```

Comment:
Wanted to check-in on the motivation behind removing this method, which is meant to keep the layer info in the `AnnotationDialog` up to date. Is the thinking that a player shouldn't be able to create new layers while the dialog is open? because this definitely can happen given we're using a modeless dialog that doesn't lock other UI actions from occuring while it's visible.

Reply:
I removed it because the dialog never actually updated itself when the new layers were set, so it was just a noop. If you think its better to have it stay up-to-date, I can restore this and make sure that it fully updates

Reply:
Gotcha! We probably should update the layers in the dialog if a player creates one while the dialog is open. I can definitely imagine that happening enough to warrant it